### ISSUE TITLE
 decay is deprecated in the new Keras optimizer

### ISSUE THREAD

**Issue by nithinjp1997** [jasmcaus/canaro#2]
The optimizer used in canaro is SGD but the new SGD optimizer doesn't support decay. Please change it into the legacy SGD optimizer tf.keras.optimizers.legacy.SGD.

**Comment by MAHENDRA9535** [comment]
> The optimizer used in canaro is SGD but the new SGD optimizer doesn't support decay. Please change it into the legacy SGD optimizer tf.keras.optimizers.legacy.SGD.

for developers who want a instant fix,
#import all these modules
import tensorflow as tf
import keras
from keras.models import Sequential, Model
from keras.layers import Conv2D, Flatten, MaxPooling2D, Dense, Input, Reshape, Concatenate, GlobalAveragePooling2D, BatchNormalization, Dropout, Activation, GlobalMaxPooling2D
from keras.utils import Sequence

#function that u need to create for the model:
def create_ST_layer(input_shape = (64, 128, 3)):
    input_img = Input(shape=input_shape)
    model = Conv2D(48, kernel_size=(5, 5), input_shape = input_shape, strides = (1, 1), activation = "relu")(input_img)
    model = MaxPooling2D(pool_size=(2, 2), strides = (2, 2))(model)
    model = Conv2D(32, kernel_size=(5, 5), strides = (1, 1), activation = "relu")(model)
    model = MaxPooling2D(pool_size=(2, 2), strides = (2, 2))(model)
    model = Dense(50, activation = "relu")(model)
    model = Dense(6)(model)
    model = tf.keras.Model(inputs=input_img, outputs= model)
    return model

#print thesummary of the model,

model = create_ST_layer()
model.summary()
